modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -53,47 +63,47 @@ contains the `install-config.yaml` file you created.
 Because you create your own compute machines later in the installation process,
 you can safely ignore this warning.
 
+ifdef::aws,gcp[]
```

Comment:
@jianlinliu, we didn't have a step to remove these manifest files for vSphere or bare metal.

Reply:
After I took another look at the files that I'd need to change to apply this step to vSphere and bare metal, I realized that it would be easier to do it on this PR. For these methods, the [4.1 steps (eg vSphere)](https://docs.openshift.com/container-platform/4.1/installing/installing_vsphere/installing-vsphere.html#installation-generate-ignition-configs_installing-vsphere) skip generating the manifests and just have you generate the Ignition configs. With this change, you need to generate the manifests too ([proposed 4.2 change](https://bz1738456--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-user-infra-generate-k8s-manifest-ignition_installing-vsphere)).

The way the conditionals are set up, the 4.2 docs still won't make you remove the control plane or compute manifests.